_posts/blog/2018-09-20-camille-garcia-digital-archives-review.md

```diff
@@ -0,0 +1,17 @@
+
+The search results for episodes make it very clear to understand what each episode is about and whether it will be useful to your research. You are presented with the name of the episode (linked to the actual episode and its landing page), its categorized “Theme” and “Series,” the names of the host and interviewees, the broadcast date, and a brief description of the episode. Click on an episode to read a more detailed description of the episode’s discussion and to listen to or download the episode, if you wish.
+
+Overall, this digital archive is an easy-to-use, interesting gateway to literally hear stories from the past, many of which are still relevant today and range from curanderismo and Tejano music to education reform and politics. As a researcher, it is key to note that the vast majority of the hosts and producers were Mexican-American and women and the guests featured were almost always Mexican-American, Mexican, indigenous, women, or other members of the broader Latin American community. Some guests included famous Conjunto musician Flaco Jiménez and Chicana writer Gloria Anzaldúa, who along with other community members candidly shared their personal experiences and expertise with listeners across Texas and the Southwest United States. In this sense, the creation of the Onda Latina Collection is a way to literally give voices back to marginalized populations of the time instead of leaving those voices to collect dust as outdated tapes, leaving “history” to be solely defined by what was printed in mass media then. Still, we must recognize the impact the of being beholden to the University of Texas has had on the collection, since the shows were produced, edited, and eventually archived as a university project.
+
+As Kathryn Burns wrote in Into the Archive: Writing and Power in Colonial Peru, “We historians are fond of regarding archives as windows on the past, or repositories of long-ago ‘voices,’” but if we critically analyze an archive, we often can notice silences and lacks of transparency (Burns 2010, 16-17). This archive is a good place to start comparative research of how larger media outlets and these smaller, community-based outlets reported important Mexican-American issues, keeping in mind, as Burns wrote, whose truth is actually being recorded (Burns 2010, 4).
```

Comment:
Put this book title in italics. You might also consider linking to more information about the book (the publisher's website, for example.)